### ISSUE TITLE
Need to remove confusion around the README.md file

### ISSUE THREAD

**Issue by eviltoylet** [bebraw/reactabular#40]
There's the README.md in GitHub and the one that's actually published on GitHub pages. We need to start considering a strategy where people can see the README that corresponds with the version that they're using. 

@bebraw thoughts?

**Comment by bebraw** [comment]
That's a very good point.

What if we had a separate `dev` branch? All PRs should probably go against that. `master` would contain only stable code (merge from `dev`).

We could go into a branched model after the next stable release.

**Comment by eviltoylet** [comment]
Yeah, I like that idea. Once code is merged into master, the whole publishing workflow should be run.

**Comment by bebraw** [comment]
Set up a `dev` branch. Not much there yet, though.
